src/Java.Interop.Tools.Maven/IPomResolver.cs

```diff
@@ -0,0 +1,8 @@
+using Java.Interop.Tools.Maven.Models;
+
+namespace Java.Interop.Tools.Maven;
+
+public interface IPomResolver
```

Comment:
Naming? Why not name this `IProjectResolver`, as it (currently) only resolves `Project`s?  Is it so we can resolve other things in the future while reusing the same abstraction?

Reply:
`Pom` and `Project` are synonyms, standardized on `Project`.